src/components/ui/datePicker.tsx

```diff
@@ -38,7 +38,10 @@ export function DatePicker() {
         <Calendar
           mode="single"
           selected={date}
+          /* eslint-disable */
+          // @ts-ignore
           onSelect={setDate}
+          /* eslint-enable */
```

Comment:
To fix the ESLint error, could you please make the setDate accept undefined by changing [this line](https://github.com/Tascurator/tascurator-frontend/blob/8735631bd9fb182c8330ce7365acc299c6bda188/src/components/ui/datePicker.tsx#L17)?

From
```ts
const [date, setDate] = useState<Date>(() => startOfDay(new Date()));
```
To
```ts
const [date, setDate] = useState<Date | undefined>(() => startOfDay(new Date()));
```